star-ros2/src/star_perception/star_perception/hailo_runner.py

```
@@ -0,0 +1,390 @@
+            with network_group.activate(ng_params):
+                with hp.InferVStreams(network_group,
+                                      input_vstreams_params,
+                                      output_vstreams_params) as pipeline:
+                    feed = {input_info.name: np.expand_dims(canvas, axis=0)}
+                    outputs = pipeline.infer(feed)
+            return _decode_hailo_yolov8_output(outputs, input_size)
+
+        return backend
+
+
+def _decode_hailo_yolov8_output(outputs: dict, input_size: int
```

Ruff (0.15.10)

[warning] Unused function argument: `input_size`

(ARG001)